packages/web3torrent/src/utils/checkTorrentInTracker.ts

```diff
@@ -1,33 +1,29 @@
 import {Client} from 'bittorrent-tracker';
```

Comment:
Should we rename this file so that it follow the kebab-case convention? For example, `check-torrent-in-tracker.ts`?

Reply:
oh, good catch! will do

Reply:
https://github.com/statechannels/monorepo/pull/1490/commits/3952c25531b950f53d5f9031024f668171ad4b2c